projects/05-wordpress-plugin/README.md

```diff
@@ -0,0 +1,255 @@
+# Plugin para WordPress
+
+## Índice
+
+* [1. Preámbulo](#1-preámbulo)
+* [2. Resumen del proyecto](#2-resumen-del-proyecto)
+* [3. Objetivos de aprendizaje](#3-objetivos-de-aprendizaje)
+* [4. Consideraciones generales](#4-consideraciones-generales)
+* [5. Criterios de aceptación mínimos del proyecto](#5-criterios-de-aceptación-mínimos-del-proyecto)
+* [6. Pistas, tips y lecturas complementarias](#6-pistas-tips-y-lecturas-complementarias)
+
+***
+
+## 1. Preámbulo
+
+Los sistemas de gestión de contenidos o CMS (Content Management Systems) son
+aplicaciones web que permiten a usuarios (sin necesidad de tener conocimientos
+previos en programación)  crear contenidos con una interfaz visual de manera
+rápida y sencilla. Entre los más utilizados están [WordPress(42.1%),
+Shopify(3.7%), Joomla(2.0%), Squarespace(1.7%) y
+Wix(1.7%)](https://w3techs.com/technologies/overview/content_management).
+
+WordPress es uno de los CMS más utilizados en el mercado. Éste permite crear
+blogs personales u otro tipo de sitios web, como páginas corporativas, revistas,
+tiendas online, periodicos digitales, etc(aquí te dejamos
+[WordPress Website Showcase](https://wordpress.org/showcase/)
+donde podrás encontrar otros casos de uso). Además WordPress es un
+sistema de software libre, gratuito, modular, y compatible con muchos plugins
+y temas, constantemente actualizado y con mejoras gracias a la comunidad de
+desarrolladores. El lenguaje en el que desarrollamos WordPress es PHP,
+que es el lenguaje de código abierto [más popular desde el lado del servidor (79.2%)](https://w3techs.com/technologies/overview/programming_language)
```

Comment:
Aquí también agregaría la fuente, pues "el más popular" siempre es algo debatible jeje, de hecho el estudio de [StackOverflow del 2020](https://insights.stackoverflow.com/survey/2020#technology-programming-scripting-and-markup-languages) por ejemplo pone a PHP en 8vo lugar.

```suggestion
que es el lenguaje de código abierto [más popular desde el lado del servidor (79.2%)](https://w3techs.com/technologies/overview/programming_language), según un estudio de W3Techs.com.
```